package.json

```diff
@@ -17,17 +18,16 @@
   },
   "homepage": "https://github.com/EventMobi/eslint-config-eventmobi#readme",
   "dependencies": {
-    "eslint-config-airbnb": "11.1.0",
-    "eslint-config-airbnb-base": "7.1.0"
+    "eslint-config-airbnb": "14.0.0",
+    "eslint-config-airbnb-base": "11.0.1"
   },
   "peerDependencies": {
-    "eslint": "^3.5.0",
-    "eslint-plugin-jsx-a11y": "^2.1.0",
-    "eslint-plugin-import": "^1.6.0",
-    "eslint-plugin-react": "^6.0.0"
+    "eslint": "^3.13.0"
   },
   "devDependencies": {
-    "eslint": "^3.5.0",
-    "eslint-plugin-import": "^1.13.0"
+    "eslint": "^3.13.0",
```

Comment:
wondering if this will also fix our eslint protractor support. Because the fix is in v3.13 (https://github.com/eslint/eslint/issues/3457). Do you think we need to do anything else to add protractor support in eslint?